frontend/src/__tests__/integration/pages/clusterSettings/ClusterSettings.stories.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/jsx-no-constructed-context-values */
```

Comment:
Disabling rules for in storybook stories rather than fixing as eventually these stories will be gone sooner than later.